README.md

```diff
@@ -152,6 +153,13 @@ Export the deduplicated dataset to a file (`output.csv`)
 asreview data dedup MY_DATASET.ris -o output.csv
 ```
 
+The PID is case sensitive so if the column header in your .csv is 'DOI', use `--pid DOI`.
+This also allows for custom PIDs.
+
+```
+asreview data dedup MY_DATASET.ris -o output.csv --pid custom_PID
```

Comment:
Pubmed example? Or any other use case? (Only if possible)

Reply:
Added. I am still doubting whether to add that this specifically goes for .csv files with an appropriate column header, although I don't think anyone with a .ris will find himself in the position of thinking it would work.

Reply:
Yes, this example is, in general, limited to tabular files. It might be better to use a tabular format in the example.